lib/libvfio-user.c

```diff
@@ -1587,7 +1582,7 @@ vfu_addr_to_sg(vfu_ctx_t *vfu_ctx, vfu_dma_addr_t dma_addr,
 }
 
 EXPORT int
-vfu_map_sg(vfu_ctx_t *vfu_ctx, const dma_sg_t *sg,
+vfu_map_sg(vfu_ctx_t *vfu_ctx, dma_sg_t *sg,
```

Comment:
> on one line now?

right

Reply:
> since we're changing the API, should we take the opportunity to add a flags argument maybe?
I'm thinking of future vfu_mark_dirty(). It can just take only 0 for now

Good idea.